// Organizing proto files (recommended by Google engineer)
// https://github.com/protocolbuffers/protobuf/issues/1491#issuecomment-1022571406

syntax = "proto3";

import "protobufs/model.proto";

package invocation.v1;

message InvocationRequest {
    model.v1.ModelInput model_input = 1;
}

message InvocationResponse {
    string status = 1;
    string message = 2;
}

service Invocation {
    rpc Invoke (InvocationRequest) returns (InvocationResponse);
}
